source/api_reference/index.html.md.erb

```diff
@@ -150,7 +150,7 @@ Common status codes are:
 |--------------|------------|---------|-------|
 | Create payment | P0101 | Missing mandatory attribute | The request you sent is missing a required attribute. |
 | Create payment | P0102 | Invalid attribute value | The value of an attribute you sent is invalid. |
-| Create payment | P0197 | Unable to parse JSON | The JSON you sent in the request body is invalid. |
+| Create payment | P0197 | Unable to parse JSON | The JSON you sent in the request body is invalid. Check the JSON in your request body, or [contact us about Mail Order Telephone Order (MOTO) payments](/optional_features/moto_payments/) if you've used `moto: true` in your request.|
```

Comment:
I wouldn't make any changes to P0197. There should be a new row with P0196 instead: https://github.com/alphagov/pay-publicapi/blob/master/src/main/java/uk/gov/pay/api/model/PaymentError.java#L22

Reply:
Sorry Oz, I spotted my mistake and corrected it just before you commented! Fixed in https://github.com/alphagov/pay-tech-docs/pull/381/commits/166aa4228ca517d9e8b29a520910499eeede697c